### ISSUE TITLE
 Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (2 - Scoring)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#444]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

* Outcome ratings vs. critical errors vs. conformance levels is confusing.  Adding to the complexity is the objective pass/fail outcome vs adjectival ratings, making it even harder to follow. Understanding and meeting the outcomes is hard enough for people to learn that we don’t want to see there be a complex scoring added on top.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
